### PR TITLE
Add mods for multiple Minecraft versions, separate 1.8.9 from 1.7.10

### PR DESCRIPTION
**Pull request by Coccocoahelper**
**Additions:**
`- Add Phosphor Legacy Forge`
`- Add Phosphor Legacy (Fabric)`
`- Add Recommended Performance 1.8.9`
`- Add Babric b1.7.3 performance mods`

**Fixes:**
`- Fix a very old "Github" typo`
`- Fix multiple mods getting incorrectly labeled as All Rights Reserved`

### CODE REVIEW

File: Performance/PerformanceOld.md [hunk 1]
```diff
@@ -15,54 +15,55 @@ We have partnered with BisectHosting this is an exciting step for us. All money
...
+| [FoamFix](https://modrinth.com/mod/foamfix) [![Modrinth Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Modrinth.png)](https://modrinth.com/mod/foamfix)[![CurseForge Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/foamfix-optimization-mod)[![GitHub Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/asiekierka/FoamFix) | None | FoamFix is a mod designed to optimize post-1.7.10 modded Minecraft using simple, targeted optimizations. | asiekierka | Both | None | [GPL 3](/licenses/Licenses.md#gpl-3)
+| [Patcher](https://sk1er.club/mods/patcher) [![GitHub Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/Sk1erLLC/Patcher) | None | Patcher is a 1.8.9 Forge mod created to improve your all-around Minecraft experience, bringing tons of Quality of Life additions, performance improvements, and tons of Vanilla bug fixes. | Sk1er | Client | None | [CCA-NC-SA-4.0](/licenses/Licenses.md#all-rights-reserved)
```
Comment (lrmtheboss) [inline Performance/PerformanceOld.md:22]: You updated the license name but forgot to update the license link.

**Review by lrmtheboss — CHANGES_REQUESTED** [review]
One small issue, check my comments below. Also please update the description and name of this PR to accurately represent what your changes are.

File: Performance/PerformanceOld.md [hunk 2]
```diff
@@ -1,68 +1,21 @@
...
-| [Mipmaplevel and Language Fix](https://modrinth.com/mod/mipmaplevelandlanguagefix) [![Modrinth Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Modrinth.png)](https://modrinth.com/mod/mipmaplevelandlanguagefix)[![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/KingContaria/Mipmaplevel-and-Language-Fix) | Unknown | Makes changing Mipmaplevels and Language faster, by instead of doing a full reload, only reloading the respective part of the game. | KingContaria | Client | None | [CC0](/licenses/Licenses.md#cc0)
-| [SaltLib](https://github.com/DeexCoding/SaltLib) [![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/DeexCoding/SaltLib) | Unknown | SaltLib aims to be an alternative of the Soduim mod for Minecraft, but for legacy versions (1.3.2-1.13.2) of the game. | DeexCoding | Client | None | [Custom](https://github.com/DeexCoding/SaltLib/blob/1.8.9/LICENSE.txt)
-
-## Legacy Fabric 1.7.X
+## Babric b1.7.3
```
Comment (lrmtheboss) [inline Performance/PerformanceOld.md:15]: Babric? You mean Fabric?

**Review by lrmtheboss — CHANGES_REQUESTED** [review]
Just a couple small things

**Comment by Coccocoahelper** [comment]
@lrmtheboss Should be ready.

File: Performance/PerformanceOld.md (hunk 2 above)
Comment (Radk6) [reply Performance/PerformanceOld.md:15]: > Babric? You mean Fabric?

Babric is a fork of Fabric (or Legacy Fabric?) for b1.7.3

**Comment by Coccocoahelper** [comment]
> > Also please update the description and name of this PR to accurately represent what your changes are.
> 
> Still need to do this, a title of "Add Mods" is too generic
> 
> > not everything, yet! there's a lot more...
> 
> The description does not tell what you added even in a generic form like "Added mods for Babric for Minecraft version b1.7.3" You should also include other changes like how you made 1.8 it to separate pages from the generic older pages.

Thanks for the nicely delivered suggestions! I'll do that

**Review by lrmtheboss — APPROVED** [review]
LGTM

**Pull request merged**